clients/mobile/App.tsx

```diff
@@ -1,16 +1,26 @@
+import 'react-native-gesture-handler'
 import React, { useEffect, useState } from 'react'
 import { BackHandler, ToastAndroid } from 'react-native'
-import 'react-native-gesture-handler'
 import { SafeAreaProvider } from 'react-native-safe-area-context'
 import { NavigationContainer } from '@react-navigation/native'
 import { ThemeProvider } from './src/setup/theme/context/ThemeContext'
 import Navigator from './src/setup/navigation/Navigator'
 import { Amplify } from 'aws-amplify'
 import { awsCognitoConfiguration } from './src/setup/config/cognito'
 import { AuthProvider } from './src/authentication/context/AuthContext'
+import { NotificationProvider } from './src/setup/notification/NotificationProvider'
+import * as Notifications from 'expo-notifications'
 
 Amplify.configure(awsCognitoConfiguration)
 
+Notifications.setNotificationHandler({
```

Comment:
are you sure you want to keep it here? also why is it async?

Reply:
I kept intentionally on App with a thought that, the mobile notification system will be ready at the very beginning op app running. This may ensure that any incoming notification maybe while app is not running in back end are handled from the very beginning of the app life cycle.
Actually my implementation may not using any async operation. I followed the system of `handleNotification` from an example of `expo-notification`. Async was used here so that it doesn't prevent other setups to load on app startup.

Reply:
Got your point. But will there be any issue/bad side if it stays like this? If it does then I surely will rewrite this part.